types/express/index.d.ts

```diff
@@ -0,0 +1,7 @@
+import { User } from '../../src/user/schemas/user.schema'
+
+declare module 'express-serve-static-core' {
```

Comment:
```suggestion
declare module 'express' {
```
For clarity, it's better redeclaring those modules from which you'll import types (= their public interfaces).

Reply:
Fixed